oyelakinG9/Hello-Rails-App
# Create Pages Rail File

# Hello Rails app

- Let's use Rails' generators (known as scaffolding) to automagically make the files we need. For this you will do rails generate controller Pages hello in your shell, look at the example:
$> rails generate controller Pages hello

    create  app/controllers/pages_controller.rb
     route  get 'pages/hello'
    invoke  erb
    create    app/views/pages
    create    app/views/pages/hello.html.erb
    invoke  test_unit
    create    test/controllers/pages_controller_test.rb
    invoke  helper
    create    app/helpers/pages_helper.rb
    invoke    test_unit
- Looking at the output of the command you will see all the generated things, but for now you only need to focus on one app/views/pages/hello.html.erb. You will put your HTML in this one.
- Open the file app/views/pages/hello.html.erb and add an h1 element saying "Hello world".
- Open http://localhost:3000/pages/hello in your browser. You should see your "Hello world" message.
- In config/routes.rb replace get 'pages#hello' with root 'pages#hello'
- Revisit http://localhost:3000/ and see your changes in action. Now your "Hello world" page should be displayed as the main page.
- Push all your changes to GitHub.
